src/DataProtection/AzureKeyVault/src/Microsoft.AspNetCore.DataProtection.AzureKeyVault.csproj

```diff
@@ -1,17 +0,0 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <Description>Microsoft Azure KeyVault key encryption support.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageTags>aspnetcore;dataprotection;azure;keyvault</PackageTags>
-    <IsPackable>true</IsPackable>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <Reference Include="Microsoft.AspNetCore.DataProtection" />
-    <Reference Include="Microsoft.Azure.KeyVault" />
-    <Reference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" />
```

Comment:
Do we still need the Microsoft.IdentityModel.Clients.ActiveDirectory dependency e.g. in eng/Versions.props? Please remove if possible.

Reply:
We use it in the Auth middleware.